tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Tests/JavaScriptTestRunner.cs

```diff
@@ -1,20 +1,19 @@
 using Azure.Sdk.Tools.Cli.Helpers;
-using Azure.Sdk.Tools.Cli.Models;
 
 namespace Azure.Sdk.Tools.Cli.Services.Tests;
 
 public class JavaScriptTestRunner(IProcessHelper processHelper) : ITestRunner
 {
-    public async Task RunAllTests(string packagePath, CancellationToken ct = default)
+    public async Task<bool> RunAllTests(string packagePath, CancellationToken ct = default)
```

Comment:
Instead of the boolean as return type. We should consider setting DefaultCommandResponse as the return type that way we don't have to do the translation twice? From result code -> bool -> DefaultCommandResponse